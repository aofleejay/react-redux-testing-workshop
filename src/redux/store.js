import { createStore, applyMiddleware, combineReducers } from 'redux'
import thunk from 'redux-thunk'
import commentReducer from './comment'

const reducers = combineReducers({
  comments: commentReducer,
})

const store = createStore(reducers, applyMiddleware(thunk))

export default store
