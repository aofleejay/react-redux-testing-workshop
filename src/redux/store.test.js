import store from './store'

describe('Test redux store', () => {
  it('Should return initial state correctly', () => {
    const initialStates = store.getState()
    expect(initialStates).toEqual({ comments: [] })
  })
})
