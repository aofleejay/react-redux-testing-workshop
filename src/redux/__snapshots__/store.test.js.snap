// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test redux store Should return initial state correctly 1`] = `
Object {
  "comments": Array [],
}
`;
